### PR TITLE
Prepared for authorization code flow

### PR DESCRIPTION
**Pull request by jekuaitk**
Depends on changes in https://github.com/itk-dev/openid-connect/pull/22.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,6 +58,9 @@ itkdev_openid_connect:
         # Optional: Specify leeway (seconds) to account for clock skew between provider and hosting
         #           Defaults to 10
         leeway: '%env(int:ADMIN_OIDC_LEEWAY)%'
+        # Optional: Allow http OIDC discovery document (used for mocking a IdP)
```
Comment (rimi-itk) [inline README.md]: It's not just for the discovery document, but for all (server side) requests to the IdP.

File: src/Controller/LoginController.php [hunk 2]
```diff
@@ -37,7 +37,7 @@ public function login(Request $request, SessionInterface $session, string $provi
         $session->set('oauth2state', $state);
         $session->set('oauth2nonce', $nonce);
 
-        $authUrl = $provider->getAuthorizationUrl(['state' => $state, 'nonce' => $nonce]);
+        $authUrl = $provider->getAuthorizationUrl(['state' => $state, 'nonce' => $nonce, 'response_type' => 'code', 'scope' => 'openid email profile']);
```
Comment (rimi-itk) [inline src/Controller/LoginController.php]: Adding line breaks after the commas may make this a little easier to read.

**Review by rimi-itk — COMMENTED** [review]
Looks good. A couple of comments have been added.

File: src/Controller/LoginController.php (hunk 2 above)
Comment (jekuaitk) [reply src/Controller/LoginController.php]: Agreed. Added line breaks.

File: README.md (hunk 1 above)
Comment (jekuaitk) [reply README.md]: Good point. Rephrased.

**Review by rimi-itk — APPROVED** [review]

**Pull request merged**